packages/driver/src/cypress/commands.ts

```diff
@@ -13,6 +13,76 @@ const builtInCommands = [
   addCommand,
 ]
 
+const reservedCommandNames = {
```

Comment:
Can we alphabetize these? 😬

Also, they don't need to be quoted, it can be `queue: true` instead of `'queue': true` for example.